week2-section1/exercise4/Main.cpp

```diff
@@ -0,0 +1,91 @@
+/*
+* Create a printer class. This class should allow you to store a string and then output it in different formats. Ask the user to input a string and then show what your printer can do! Remember to use char arrays instead of strings and avoid the std namespace.
+*/
+
+#include <iostream>
+#include<string>
+
+class Printer
+{
+private:
+	std::string WordEntry;
+public:
+	 Printer()
+	{
+		WordEntry = " ";
+	};
+	Printer(std::string OutputString) {
+		WordEntry = OutputString;
+	};
+	void Print() {
+		std::cout << WordEntry << std::endl;
+	};
+
+	void PrintReversed() {
+		int size = 0;
+
+		for (int i = 0; i < 30; i++)
+		{
+			if (WordEntry[i] == '\0')
+			{
+				break;
+			}
+			size++;
+		}
+
+		char wordReverse[6];
+
+		for (int i = size - 1; i >= 0; i--)
+		{
+			wordReverse[size - 1 - i] = WordEntry[i];
```

Comment:
What if the word is greater than 6 characters? Your array is of size 6, what if Word Entry is of size 30? It may actually be easier to actually use a string here.